pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -190,12 +194,13 @@ public Map<String, MercuryOrderDto> getPossibleBarcodesForParticipant(String ddp
 
     @Override
     public int create(MercuryOrderDto mercuryOrderDto) {
-        throw new IllegalStateException("This method should not be used");
+        throw new IllegalStateException("Call the version of this method that includes the raw json payload");
```

Comment:
I do wonder why we have these interfaces--a number of implementations throw exceptions or do nothing (see the delete() method below).  I'm trying to make this a little less cryptic without refactoring the universe.

Reply:
Thanks for cleaning up the error! We talked about abstract Dao implementations but  I would also vote for stopping the use of Dao interfaces in DSM. I don't think they are adding any values.